### ISSUE TITLE
Is there an option to keep trainer pokemon unchanged but to add additional pokemon to their party?

### ISSUE THREAD

**Issue by AWWAMW** [Ajarmar/universal-pokemon-randomizer-zx#578]
During my playthroughs I have been wanting to strategize against enemy trainers with looking up their teams beforehand, but I have been wanting to add a random surprise to their team in the form of a new Pokémon in their team, preferably one that fits their team already.
I think most advantageously would be, when selecting in the Foe Pokémon tab; 
Trainer Pokémon: Unchanged
That you can still additionally select; Additional Pokémon for: Boss Trainers | Important trainers | Regular trainers (plus add held items to:)
And ontop of that also Double battle mode and Better movesets for unchanged trainer Pokémon
I feel like even with selecting Trainer Pokémon: Unchanged there should still be possibilities to alter some things slightly

TLDR; Still be able to checkmark all the boxes in the FOE POKEMON tab whilst Trainer Pokémon has been put on Unchanged
Is it possible that anyone can look into this?
Thanks

**Comment by username-1194** [comment]
I agree with this guy

**Comment by agoodboyatheart** [comment]
Off the back of this because it’s related - any chance the “better movesets” option could be enabled even if foe randomisation is not enabled?

**Comment by tom-overton** [comment]
@agoodboyatheart Enabling better movesets with unrandomized trainer Pokemon was super duper easy, so I just knocked it out in eee7ace189e3d38381f2bf1a27670904931532a9. Will be in the next release.

Now as for everything else, we do not take feature requests, and we have no plans to expand the range of options available without trainer randomization *other than* the better movesets thing I just mentioned. It is quite challenging to find an additional Pokemon that "fits their team already", because it's a matter of opinion as to what "fits". In Red, Brock has two Rock/Ground Pokemon; is he a Rock trainer or a Ground trainer? What kind of Pokemon "fits" with Cynthia's team in Platinum; just any strong Pokemon? What about a random Team Magma Grunt; how is the randomizer supposed to know that these guys only really use Zubat, Poochyena, and Numel? Trying to determine what "fits" for a given unrandomized trainer might *sound*, simple, but it's actually incredibly complicated, and we aren't interested in solving this problem.
